configs/config.toml

```diff
@@ -1,20 +1,28 @@
 
 [exp]
   log_level = "debug"
-  nats = "nats://localhost:4222"
+  nats = "nats://127.0.0.1:4222"
```

Comment:
Why not `localhost`? Just curious

Reply:
no special reason

Reply:
I would keep `localhost` in this case

Reply:
@mteodor please keep `localhost`, to stay consistent to other Mainflux services.